_posts/2015-12-18-fengzhichu-theme.markdown

```diff
@@ -1,8 +1,8 @@
 ---
 layout: post
 title: Fengzhichu-Theme
-date: 2015-09-29 12:12:12
-homepage: https://github.com/fengzhichu/fengzhichu-theme/tree/gh-pages
+date: 2015-12-18 17:10:00
```

Comment:
Hey! Please don't change the date because this will make the date in front matter and file name out of sync. Don't rename the file either, just keep the date as it was. Thanks!